ui/decredmaterial/icon_gallery.go

```diff
@@ -12,7 +12,7 @@ type Icons struct {
 	ContentAdd, NavigationCheck, NavigationMore, ActionCheckCircle, ActionInfo, NavigationArrowBack,
 	NavigationArrowForward, ActionCheck, ChevronRight, NavigationCancel, NavMoreIcon,
 	ImageBrightness1, ContentClear, DropDownIcon, Cached, ContentRemove, ConcealIcon, RevealIcon,
-	SearchIcon, PlayIcon *widget.Icon
+	SearchIcon, PlayIcon, ErrorIcon *widget.Icon
```

Comment:
this is not needed as the icon used for the modal on the mockup already exist.

<img width="287" alt="image" src="https://user-images.githubusercontent.com/27733432/185954411-4649b499-998e-4484-82fd-0a1c1ec0d82d.png">